### ISSUE TITLE
Capability to copy credentials to clipboard instead of to ~/.aws/credentials

### ISSUE THREAD

**Issue by e-moshaya** [Noovolari/leapp#59]
Similar to AWS SSO Portal, it would be nice for Leapp to allow us to copy the credentials to clipboard instead of writing it to ~/.aws/credentials

**Comment by ericvilla** [comment]
Hi @e-moshaya, can you please describe the use case? Just to be clear, our idea is to write to the ~/.aws/credentials file which is the one supported by almost all the third-party tools. We are considering the possibility to write them even in the ~/.aws/cli/cache folder for further compatibility. Moreover, we're going to add support for multiple "parallel" profiles, allowing you to switch from one to another in just a click.

**Comment by e-moshaya** [comment]
Our usecase would be for people who don't wish to use the shared credentials file for all terminal/shell sessions. e.g. I could have a terminal/shell tab with credentials for account a, a second tab using credentials for account b. Having the credentials copied and pasted as environment variables AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_SESSION_TOKEN allows us to have multiple sessions running in parallel with multiple accounts

AWS SSO Portal allows you to generate these temporary credentials and copied to clipboard, see screenshots below:

![image](https://user-images.githubusercontent.com/44640934/104481690-fb67dd00-55bd-11eb-9ca2-bd71017b5b99.png)

![image](https://user-images.githubusercontent.com/44640934/104481711-03278180-55be-11eb-92b8-fe780251f88b.png)

**Comment by ericvilla** [comment]
Ebrahim, that's very interesting! That could be done through a Leapp's CLI, something that we are going to work on as described in the roadmap. It would be nice to have a command that sets these environment variables automatically! Btw, we already have something that allows you to copy credentials to clipboard, just have to consolidate the functionality.
